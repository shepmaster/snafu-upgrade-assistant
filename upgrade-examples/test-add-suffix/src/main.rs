use snafu::Snafu;

#[derive(Debug, Snafu)]
enum EnumError {
    EnumVariant1,
    EnumVariant2 { name: String },
}

#[derive(Debug, Snafu)]
struct StructError;

fn main() {
    let _ = EnumVariant1.build();
    let _ = EnumVariant2 { name: "name" }.build();
    let _ = StructContext.build();
}
