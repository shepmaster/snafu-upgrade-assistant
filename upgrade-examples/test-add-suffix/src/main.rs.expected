use snafu::Snafu;

#[derive(Debug, Snafu)]
enum EnumError {
    EnumVariant1,
    EnumVariant2 { name: String },
}

#[derive(Debug, Snafu)]
struct StructError;

fn main() {
    let _ = EnumVariant1Snafu.build();
    let _ = EnumVariant2Snafu { name: "name" }.build();
    let _ = StructSnafu.build();
}
